.github/workflows/ci.yml

```diff
@@ -4,9 +4,15 @@ on:
   - pull_request
 
 env:
-  GO_VERSION: "1.21.x"
+  GO_VERSION: "oldstable"
```

Comment:
why did we change this?

Reply:
so we don't always have to keep updating the version from 1.21 to 1.22 (when 1.23 is stable), and so one
itll give us an early warning when releases break us